src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -27,29 +28,100 @@
     private WPI_TalonSRX horizontalL = new WPI_TalonSRX(RobotMap.horizontalClimb_L);
     private AHRS ahrs = new AHRS(SPI.Port.kMXP);
     private double vp = 0.05;
-    private double hp = 0.8;
+    private double hp = 0;//0.8;
     private double vError, vResponce, hError, hResponce = 0;
 
+    private Boolean updateState = null; 
+
     private int oldLVertical, oldRVertical, oldLHorizontal, oldRHorizontal = 0;
 
     public Climber_Subsystem() {
         super();
-        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        verticalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
+        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        verticalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        horizontalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+        horizontalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
+
+    }
+    
+    //Initialization function that should be called only once
+    //This should be called before climbing but not on robot boot so manual adjustments to the climber
+    //before the game starts won't mess it up
+    public void singleInit()
+    {
+        //Hard reset the encoders to 0
```

Comment:
This is called on autonomousInit (see Robot.java) and hard resets all the encoders to 0. This means we don't have to deal with any of the soft reset stuff when we're dealing with the soft limits. Because it's run on autonomousInit it _should_ be done before we leap from the platform so any disturbance to the climber from that jump won't affect the soft limits.